scripts/test-coderabbit-review-workflow.sh

```
@@ -0,0 +1,29 @@
+assert_contains() {
+    local expected="$1"
+    if ! grep -Fq -- "${expected}" "${workflow_file}"; then
+        echo "Workflow contract missing: ${expected}" >&2
+        exit 1
+    fi
+}
+
+# This workflow intentionally uses only GitHub API shell commands and does not check out or
+# execute pull-request code. These source-contract assertions deterministically exercise the
+# security-sensitive behavior that can be validated without contacting GitHub.
+assert_contains 'if: ${{ github.event.pull_request.draft == false }}'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)